src/main/java/com/contrastsecurity/ui/com/contrastsecurity/ui/toolwindow/ContrastToolWindowFactory.java

```diff
@@ -994,6 +993,58 @@ private void populateVulnerabilityDetailsRecommendation(RecommendationResource r
         }
     }
 
+    public void parseHtmlLinksAddToPanel(JTextPane text, JPanel panel) {
+        String linkStartKey = "https://";
+        String linkEndKey = "{{{nl}}}";
+
+        String labelText = text.getText().replaceAll("<br>", "").replaceAll("<b>", "").replaceAll("</b>", "").
```

Comment:
The .NET section is missing a line break.
![image](https://user-images.githubusercontent.com/17487428/112347712-7a078780-8c9d-11eb-913f-e0b5e7a1a080.png)

Reply:
I think that might be a result of the script, but I changed the replacement for breaks to a newline to account for that.